java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -173,4 +181,26 @@ public CompletableFuture<String> set(
         String[] arguments = ArrayUtils.addAll(new String[] {key, value}, options.toArgs());
         return commandManager.submitNewCommand(SetString, arguments, this::handleStringOrNullResponse);
     }
+
+    @Override
+    public CompletableFuture<String[]> mget(@NonNull String[] keys) {
+        return commandManager
+                .submitNewCommand(MGet, keys, this::handleArrayReponse)
+                .thenApply(
+                        objectsArray ->
+                                Arrays.stream(objectsArray).map(object -> (String) object).toArray(String[]::new));
+    }
+
+    @Override
+    public CompletableFuture<String> mset(@NonNull Map<String, String> keyValueMap) {
+        String[] args = new String[keyValueMap.size() * 2];
```

Comment:
You should be able to convert the entryset to an array using `stream().flatmap()`
But I don't know the exact code.  You will need to put the key, value into another stream and then toArray at the end.